Outlook_parser.py

```diff
@@ -0,0 +1,28 @@
+import win32com.client
+import unicodecsv as csv
+import Analise_horas
+
+output_file = open("sent_emails.csv", "wb")
+output_writer = csv.writer(output_file, delimiter=";", encoding="latin2")
+
+outlook = win32com.client.Dispatch("Outlook.Application").GetNamespace("MAPI")
+outbox = outlook.GetDefaultFolder(5)
```

Comment:
Este numero `5` aqui chama-se um "magic number". O que isso quer dizer e' que este numero e' magico e nao tem significado nenhum. Vejo que nos comentarios a seguir tentas descrever o que eles querem dizer, mas ha' uma estrategia melhor.

Um [Enum](https://docs.python.org/3/library/enum.html) e' um tipo de objecto que, simplificando, mapeia nomes para numeros. Especificamente, o que podias fazer aqui era definir um enum para estas pastas de outlook:

```python
from enum import Enum

class OutlookFolder(Enum):
    ??? = 1
    ??? = 2
    ??? = 3
    ??? = 4
    OUTBOX = 5
    INBOX = 6
```

Poderias investigar para saber o que e' que sao os valores de 1 a 4, mas e' opcional. Podes so' definir `OUTBOX` e `INBOX` acho. Com este enum, podias substituir esta linha por

```python
outbox = outlook.GetDefaultFolder(OutlookFolder.OUTBOX)
```

Repara como e' que isto faz com que o teu codigo seja muito mais legivel, e nao tens que usar comentarios para explicar nada! O enum funciona como documentacao, e' optimo.

Reply:
Já adicionei o Enum, no entanto ainda só com Inbox e Outbox. 
Até ao momento ainda não consegui encontrar os indexes do outlook, vi varias estratégias online para consegui o index das folders, mas no meu caso não consegui aplicar.
Se eu fizer um for loop do outlook.GetdefaultFolder() , (que me deveria dar uma lista de todas as folders) só me dá "public folders iuri.campos@telepizza.com".
Acabei por conseguir descobrir que o 5 é Outbox por tentativa e erro. Mas vou tentar investigar mais para completar os outros numeros.

Reply:
Não estava a conseguir. Tive de adicionar o ".value", ficou assim:

outbox = outlook.GetDefaultFolder(OutlookFolder.OUTBOX.value
)

Reply:
Como nao vais ter que usar o index das outras folders isto e' perfeitamente aceitavel. Se fosse uma descoberta facil, podias adicionar ao `OutlookFolder`, mas nao sendo nao se perde muito.